### ISSUE TITLE
Storing the Message in the Database: nuv debug config didn't return "postgres_url"

### ISSUE THREAD

**Issue by diegoconti77** [nuvolaris/nuvolaris-bugs#14]
The execution of the command "nuv property get --auth" is returning the following error:
![image](https://github.com/nuvolaris/nuvolaris-bugs/assets/49821042/eb80975f-f7ee-46ff-99f4-aa887ef79691).

The Command "nuv config export" should return the needed API key.

**Comment by ghost** [comment]
@diegoconti77 normally it should be `nuv -wsk property get --auth`

**Comment by msciabarra** [comment]
You are right - it has been changed to `nuv debug status` - we have to fix documentation.
